### PR TITLE
normalize the wifi setting

### PR DESCRIPTION
**Pull request by seuros**
Normalized wifi settings across the examples.

### CODE REVIEW

File: src/docs/devices/Zemismart-Moonlamp/index.md [hunk 1]
```diff
@@ -55,7 +55,7 @@ esphome:
   board: esp01-1m
 
 wifi:
-  ssid: !secret wifissid
+  ssid: !secret wifi_ssid
   password: !secret wifipw
```
Comment (jesserockz) [inline src/docs/devices/Zemismart-Moonlamp/index.md]: Missed one?

**Comment by jesserockz** [comment]
After this PR is merged (or part of this PR), if you would like to, the entire `wifi` should be as simple as:

```yaml
wifi:
  ssid: !secret wifi_ssid
  password: !secret wifi_password
  ap:
```

**Comment by seuros** [comment]
> After this PR is merged (or part of this PR), if you would like to, the entire `wifi` should be as simple as:
> 
> ```yaml
> wifi:
>   ssid: !secret wifi_ssid
>   password: !secret wifi_password
>   ap:
> ```

When the ap is not configured, where do it get the ssid and password ?

**Comment by jesserockz** [comment]
The name of the device is automatically used. Default is no password / open network.

File: src/docs/devices/Zemismart-Moonlamp/index.md [hunk 2]
```diff
@@ -55,9 +55,8 @@ esphome:
   board: esp01-1m
 
 wifi:
-  ssid: !secret wifissid
-  password: !secret wifipw
-  reboot_timeout: 60min
+  ssid: !secret wifi_ssid
+  password: !secret wifi_password  reboot_timeout: 60min
```
Comment (jesserockz) [inline src/docs/devices/Zemismart-Moonlamp/index.md]:
```suggestion
  password: !secret wifi_password
  reboot_timeout: 60min
```

**Review by jesserockz — APPROVED** [review]

**Pull request merged**